bin/quickpkg

```diff
@@ -43,6 +46,19 @@ def quickpkg_atom(options, infos, arg, eout):
 	fix_metadata_keys = ["PF", "CATEGORY"]
 
 	try:
+		binpkg_compression = settings.get("BINPKG_COMPRESSION", "bzip2")
+		compression = _compressors[binpkg_compression]
+	except KeyError as e:
+		eout.eerror("Unsupported compression method: %s" % e.args[0])
+		del e
+		return
+
+	if subprocess.call(["type", compression["decompress"]], shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE) != 0:
```

Comment:
Okay, I'll have a look at that one. I was aware of shutil.which() but not of find_binary(). I was considering distutils.spawn.find_executable().
Do you think assuming compression["decompress"].partition(' ')[0] is a safe assumption for the binaryname?